src/database/exiles_database.py

```diff
@@ -107,27 +134,29 @@ def get_user_exiles(user_id) -> List[tuple]:
         cursor.execute(query, params)
         res = cursor.fetchall()
 
-        return res
+        if res is not None:
+            return [Exile(*row) for row in res]  # Create a list of Exile objects
+        else:
+            return None
 
 
-def get_user_active_exile(user_id) -> PendingExile:
+def get_all_active_exiles() -> Exile:
     conn = DatabaseConnection()
 
     with conn.get_cursor() as cursor:
         query = """
-        SELECT e.exileID, u.userID, u.discordUserID, e.endTimestamp
+        SELECT e.exileID, u.userID, u.discordUserID, e.reason, e.exileStatus, e.startTimestamp, e.endTimestamp
         FROM exiles e
         JOIN users u ON e.userID = u.userID
-        WHERE u.userID = %s AND  e.exileStatus = %s 
-        LIMIT 1;
+        WHERE e.exileStatus = %s AND e.reason != 'roulette';
```

Comment:
Are you planning on following up here to change this from the reason to the createdBy that we talked about? I don't think that change is a blocker just curious

Reply:
not in this ticket, i think thats better suited to another one.. especially given the limited time I have before I leave + FRU

Reply:
if we're not doing it here then can you make a ticket for it?

Reply:
Created: https://naurffxiv.atlassian.net/browse/MOD-108